git-squash-tags

```diff
@@ -8,16 +8,18 @@ if [ $# -eq 1 ]; then
 fi
 
 echo "Grabbing tags following pattern $tag_prefix..."
-tags=(`git log --reverse --pretty=%d --tags=$tag_prefix --no-walk | sed 's/ (tag: //' | sed 's/)//'`)
+tags=(`git log --reverse --pretty=%d --tags=$tag_prefix --no-walk |grep 'tag:' | sed 's/.*tag: //' | sed 's/)//'|sed 's/,.*//'`)
```

Comment:
What are these changes attempting to accomplish? Any example log output that justifies this change? Also nit missing space between `|` and `grep`.